client/templates/admin/surveyAdmin.js

```diff
@@ -0,0 +1,55 @@
+Template.surveyAdmin.events({
+    'click .survey': function() {
+        Meteor.call('populateSurvey', function(error, result) {
+            if (result) {
```

Comment:
Populates the survey from the backend and sends it to the users. Ull find the method in server.js in methods